ipynb/tests/Frequency Invariance Test.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
It might be worth changing the name of the notebook to Frequency_Invariant_Test.ipynb to avoid having spaces in a file's name.